edisgo/opf/timeseries_reduction.py

```diff
@@ -11,16 +11,15 @@
 logger = logging.getLogger(__name__)
 
 
-def _scored_critical_current(edisgo_obj, grid):
-    # Get allowed current per line per time step
-    i_lines_allowed = check_tech_constraints.lines_allowed_load(edisgo_obj, "mv")
-    i_lines_pfa = edisgo_obj.results.i_res[grid.lines_df.index]
+def _scored_critical_loading(edisgo_obj):
```

Comment:
This is not new with this PR but why do we only check lines here? Would transformers not be of interest as well?

Reply:
This is because only MV is considered.